pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -1848,6 +1862,8 @@ message KialiConfig {
 
   // See PodAntiAffinityLabelSelector.
   TypeSliceOfMapStringInterface podAntiAffinityTermLabelSelector = 13 [deprecated=true];
+
+  TypeSliceOfMapStringInterface tolerations = 14 [deprecated=true];
```

Comment:
Is this actually supported in the kiali chart?

Reply:
added another PR in installer to support that